### PR TITLE
class for readme length in words

### PR DESCRIPTION
**Pull request by dja1443**

### CODE REVIEW

File: src/metric.ts [hunk 1]
```diff
@@ -109,29 +112,101 @@ export class NetScore extends Metric {
...
 
   constructor(repoOwner: string, repoName: string) {
     super(repoOwner, repoName);
     this.name = "RampUp";
     this.value = 0;
+
+    //this.octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
   }
 
-  // Calculate RampUp
```
Comment (Joedahms) [inline src/metric.ts:122]: Delete all this commented out code.

File: src/metric.ts [hunk 2]
```diff
@@ -109,29 +112,101 @@ export class NetScore extends Metric {
...
+      this.logger.add(2, "Error fetching readme: " + error);
+    }
+  }
+}
+
+
 // RampUp metric
 export class RampUp extends Metric {
   value: number;
+  //private octokit: Octokit; // Octokit object for API access
```
Comment (Joedahms) [inline src/metric.ts:165]: Delete this

File: src/metric.ts [hunk 3]
```diff
@@ -109,29 +112,101 @@ export class NetScore extends Metric {
...
 
   constructor(repoOwner: string, repoName: string) {
     super(repoOwner, repoName);
     this.name = "RampUp";
     this.value = 0;
+
+    //this.octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
```
Comment (Joedahms) [inline src/metric.ts:172]: Delete this

File: src/metric.ts [hunk 4]
```diff
@@ -109,29 +112,101 @@ export class NetScore extends Metric {
   }
 }
 
+
```
Comment (Joedahms) [inline src/metric.ts:115]: This should be a function in api_access.ts. I'm aiming to make all API calls in api_access.ts, keeping them out of metric.ts. Metric.ts is just for calculating the actual metrics. When you move it to api_access.ts, see if you can change it from a class to function.

**Review by Joedahms — CHANGES_REQUESTED** [review]
See comments.

**Review by dja1443 — COMMENTED** [review]
Done

**Pull request closed without merging**